iris_grib/_load_convert.py

```diff
@@ -844,22 +888,37 @@ def grid_definition_template_20(section, metadata):
     metadata['dim_coords_and_dims'].append((x_coord, x_dim))
 
 
-def _calculate_proj_coords_from_lon_lat(section, cs):
+def _calculate_proj_coords_from_grid_lengths(section, cs):
     # Construct the coordinate points, the start point is given in millidegrees
     # but the distance measurement is in 10-3 m, so a conversion is necessary
     # to find the origin in m.
 
+    MM_TO_M = 1e-3
```

Comment:
It's a small thing, but by convention a variable named in all-caps is used to indicate a global variable. As this is defined in a non-global context, perhaps you could make this variable name lower-case.